atflatcontrols/atlistbox.pas

```diff
@@ -380,7 +403,9 @@ procedure TATListbox.DoDefaultDrawItem(C: TCanvas; AIndex: integer; R: TRect);
         );
 
       Inc(NColOffset, NColWidth);
+      {$ifdef FPC}
       C.Line(NColOffset-1, R.Top, NColOffset-1, R.Bottom);
+      {$endif}
```

Comment:
add {$else} with MoveTo, LineTo - it is painting of column separator line

Reply:
?: 
      C.MoveTo (NColOffset-1, NColOffset-1);
      C.LineTo (R.Top, R.Bottom);

Reply:
No, read the docs about MoveTo